react/react_events/README.md

```diff
@@ -31,7 +33,7 @@ Enter **state**. State is a way of storing information in a component and render
 This is a very simple app to count up from zero. Let's take a look at what's going on here.
 
 
-The first thing you might notice is our constructor function:
+The first thing you might notice in our class component is our constructor method. A constructor method is a special method of a class for creating and initalizing data for that class: 
```

Comment:
Do we need to take even more of a step back here and explain that our class component extends the `React.Component` class? I don't want to open up a rabbit hole but I think that will help give some more context about why we call `super()` and what is meant by `the parent's constructor` below.

Reply:
I think we should, good call